install/kubernetes/cilium/charts/agent/templates/svc.yaml

```diff
@@ -17,3 +17,45 @@ spec:
   selector:
     k8s-app: cilium
 {{- end }}
+{{- if .Values.global.hubble.hubbleUi.enabled }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: hubble-grpc
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    k8s-app: hubble
+  ports:
+  - targetPort: 50051
+    protocol: TCP
+    port: 50051
+{{- end }}
+{{- if .Values.global.hubble.hubbleUi.enabled }}
```

Comment:
Same as above, the `ServiceMonitor` and its related service are not dependent on Hubble UI.

Reply:
Addressed review comments.